src/components/editor/property/InputLiteral.jsx

```diff
@@ -109,17 +137,19 @@ const mapStateToProps = (state, ownProps) => {
   const displayValidations = getDisplayResourceValidations(state)
   const formData = findNode(state, reduxPath)
   const errors = findResourceValidationErrorsByPath(state, reduxPath)
+  const diacriticMode = state.selectorReducer.editor.diacritics.show
```

Comment:
Calling this diacriticMode when a boolean is confusing. Perhaps shouldShowDiacritic or displayDiacritic?

Reply:
Done, renamed to `shouldShowDiacritics`.